src/EnergyPlus/EvaporativeFluidCoolers.cc

```diff
@@ -1361,9 +1360,41 @@ namespace EvaporativeFluidCoolers {
         int PltSizCondNum = state.dataPlnt->PlantLoop(this->plantLoc.loopNum).PlantSizNum;
         if (PltSizCondNum > 0) {
             this->DesignExitWaterTemp = state.dataSize->PlantSizData(PltSizCondNum).ExitTemp;
-            if (this->DesignEnteringWaterTemp == DataSizing::AutoSize) {
+        }
+
+        if (this->DesignEnteringWaterTempWasAutoSized && this->PerformanceInputMethod_Num == PIM::UserSpecifiedDesignCapacity) {
+            if (PltSizCondNum > 0) {
```

Comment:
If we're in UserSpecifiedDesignCapacity, and DesignEnteringWaterTempWasAutoSized